hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2841,4 +2841,18 @@
     </description>
   </property>
 
+
+  <property>
+    <name>ozone.scm.ca.list.retry.wait.duration</name>
```

Comment:
NIT: Can we rename ozone.scm.ca.list.retry.wait.duration to ozone.scm.ca.list.retry.interval to be consistent with other retry settings?

Reply:
Done